03 Misc/05 Commons Chunk Plugin/src/students.ts

```diff
@@ -0,0 +1,12 @@
+import {getAvg} from "./averageService";
+import * as $ from 'jquery';
+import leftPad = require('left-pad');
```

Comment:
Also works a bit on explaining people that using `import ... from ...` is not working in all cases. In fact I have planned some talk about that.